graphql_server/websockets/transport_ws_protocol/types.py

```diff
@@ -5,6 +5,8 @@
 except ImportError:
     from typing_extensions import TypedDict
 
+from dataclasses import dataclass, asdict
+
 from .contstants import (
     GQL_CONNECTION_INIT,
     GQL_CONNECTION_ACK,
```

Comment:
These constants should probably be an ENUM, but fixing that shouldn't be part of this PR